drop/modules/mae-pipeline/MAE/Results.R

```diff
@@ -77,12 +78,20 @@ res[, c('aux') := NULL]
 # Bring gene_name column front
 res <- cbind(res[, .(gene_name)], res[, -"gene_name"])
 
+# Calculate variant frequency within cohort 
+maxCohortFreq <-snakemake@params$maxCohortFreq
+res[, N_var := .N, by = .(gene_name, contig, position)]
+res[, cohort_freq := N_var / uniqueN(ID)]
+
+res[,rare := (rare | is.na(rare)) & cohort_freq <= maxCohortFreq] 
+
 # Add significance columns
 allelicRatioCutoff <- snakemake@params$allelicRatioCutoff
 res[, MAE := padj <= snakemake@params$padjCutoff &
-      (altRatio >= allelicRatioCutoff | altRatio <= (1-allelicRatioCutoff))] 
+      (altRatio >= allelicRatioCutoff | altRatio <= (1-allelicRatioCutoff)) &
+      rare
```

Comment:
include the rare column (influenced by gnomAD and cohort_freq) to make it into the final results

Reply:
remove rare